CHANGELOG.md

```diff
@@ -32,6 +32,22 @@
     end
     ```
 
+- Support `Sentry::Transaction#set_measurement` [#1838](https://github.com/getsentry/sentry-ruby/pull/1838)
+
+    Usage:
+
+    ```rb
+    # enable this feature first
+    Sentry.init do |config|
+      # other configs
+      config.custom_measurements = true
```

Comment:
ah sorry should have been clearer, I think we don't even need a config anymore, it's just an additional feature on the transaction that we support from now on.

Reply:
I see. JFYI `sentry-python` still [hides it behind a config](https://github.com/getsentry/sentry-python/blob/0233e278f36a8810ef92dc79e5e574d3dec93580/sentry_sdk/tracing.py#L731-L735).

Reply:
yep it's on my todo list to remove that